tests/conftest.py

```diff
@@ -589,8 +590,12 @@ def fake_fgs_composite(
         False
     )
 
```

Comment:
I assume this is to be invoked separately from future tests and this commented out code can be removed?